user/common-build-problems.md

```diff
@@ -568,3 +568,11 @@ Approximate available disk space is listed in the [build environment overview](/
 
 The best way to find out what is available on your specific image is to run `df -h` as part of your build script.
 If you need a bit more space in your Ubuntu builds, we recommend using `language: minimal`, which will route you to a base image with less tools and languages preinstalled. This image has approximately ~24GB of free space.
+
+## Uploading Artifacts to Sonatype
+
+When publishing via the `nexus-staging-maven-plugin` to Sonatype OSS Repository, IP addresses used by TravisCI change due to our [NAT layer](https://blog.travis-ci.com/2018-07-23-the-tale-of-ftp-at-travis-ci). To get around this, please use a `stagingProfileId` as described [here](https://travis-ci.community/t/sonatype-deployment-problems/1353/2?u=mzk). 
```

Comment:
Hey @muizzk this is great stuff! Thanks!

If possible, please avoid using `here` for links. It doesn't work for screen readers, and is worse for SEO.

In this example you could use something like:

>To get around this, please [use a `stagingProfileId`](https://travis-ci.community/t/sonatype-deployment-problems/1353/2?u=mzk).

Reply:
@plaindocs thanks, didn't know about that. I opened https://github.com/travis-ci/docs-travis-ci-com/pull/2307 to correct it. Thanks again